documentation/projects/proposals/additional_search_views/20230719-implementation_plan_additional_search_views.md

```diff
@@ -111,35 +118,156 @@ for fuzzy matches.
 **Finally**, we apply the ranking and sorting parameters, and "highlight" the
 fields that were matched in the results.
 
-The new search controller should allow for using different filters for the first
-step and to not use the full-text search. We should also create a new serializer
-for collection search requests. It should include the common parameters for
-`list` requests, such as `page` and `page_size`, and the parameters for the
-exact matches: `tag`, `creator` and `source`.
+The search controller needs to be updated to extract the first 2 steps into
+`build_search_query` method. A new `build_collection_query` should be added and
+used when the `collection` parameter is present. This method should create a
+filter query for the relevant field and value.
+
+The pagination and dead link cleanup should be the same for additional search
+views as for the default search views.
+
+#### Search request serializer updates
+
+_The initial version of this plan proposed to add new API endpoints and use path
+parameters for the values (e.g., `/image/tag/cat`) to make the URLs more
+readable, easier to share, will be easier to cache or perform cache invalidation
+required by [#1969](https://github.com/WordPress/openverse/issues/1969).
+However, since tags and creator names contain characters that are special for
+path segments (`/`, `?` and `&`) that cannot be properly encoded, this approach
+turned out to be not feasible._
+
+```{note}
+The new query parameters (`collection` and `tag`) will have `unstable__` prefix until this project is
+launched. In the text below, for brevity, the prefix is omitted.
+```
+
+The collections will use the `search` endpoint with `collection` query parameter
+set to `tag`, `creator` or `source`, and the relevant query parameters. The
+`collection` parameter validator will check that the request contains the
+necessary additional parameters: `source` for `collection=source`, `creator` and
+`source` for `collection=creator`, and `tag` for `collection=tag`. The value of
+the `source` parameter will also be checked to be an existing source using the
+provider store.
+
+For the tag route, the new singular `tag` parameter should be used, rather than
+the existing plural `tags` should be used for legibility since we are presenting
+a single tag.
```

Comment:
```suggestion
For the tag route, the new singular `tag` parameter should be used, rather than
the existing plural `tags`, since we are only presenting a single tag.
```